conan/tools/cmake/toolchain/blocks.py

```diff
@@ -824,14 +828,18 @@ def context(self):
 
         system_name, system_version, system_processor = self._get_cross_build()
 
+        cmake_sysroot = self._conanfile.conf.get("tools.cmake.cmaketoolchain:cmake_sysroot")
```

Comment:
We have ``cpp_info.sysroot``, aren't we processing it anymore?

Reply:
Ok, yes, lets use the ``conf`` and forget about ``cpp_info.sysroot``, sounds good @lasote?

Reply:
The only thing, maybe better generalize the name to something like ``tools.buid:sysroot``, so it makes sense for autotools and other build systems.